qml/pages/HomePage.qml

```diff
@@ -29,32 +29,38 @@ Page
 				title: "Codes"
```

Comment:
I have to admit it's been a while since I coded using Silica components, so please feel free to enlighten me on some of the design decisions here :)

* Why have you decided to remove the `BackgroundItem` component and roll your own logic for the context menu?
* Why is the list view no longer `page.width`? Isn't it supposed to fill the entire page?
* Why is the sizing of the `Rectangle` and the `Label` redone?

Also, the indentation is a bit all over the place :)

Reply:
- Mainreason is that the contextmenu didn't function properly. When the contextmenu is closed, the height of the contextmenu is animated to zero. In the original code, the height was immediately 0 and that resulted in a contextmenu over the listitem. The new implementation is based on the docs and has the correct animation behavior. 
- don't know, error?
- because of the first point.

Reply:
Ok, fair enough!